### PR TITLE
CJA-44 Unit 5 add met images extending db

### PR DESCRIPTION
**Pull request by reuelcj**
## What It Does

Adds a user profile image from The Met’s collection based on the user’s first name. 

Uses async and await to fetch images.

## How To Test
Load site and create a new user and view images.

## Checklist

Under penalty of public shaming, I avow that I:

- [ x] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [ ] Verified that there are no compiler or linter errors or warnings
- [ ] Verified the build in production(optimized) mode
- [ ] Updated the docs, if necessary
- [ x] Included the appropriate labels
- [ x] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
- [x ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Internet Explorer

## Dependencies

- [x] https://github.com/reuelcj/christa-reuel-ux-aop/pull/28 and https://github.com/reuelcj/christa-reuel-ux-aop/pull/29 should be resolved/merged before this is merged

### CODE REVIEW

**Comment by reuelcj** [comment]
Ready for review

File: src/api/get-profile-image.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ function randomImg(imgIds) {
 
 export default async function (name) {
   const collection = await getData(collectionIdsUrl(name));
-  //[291826] = 'unknown'. It is used here as a profile for any name that returns no images.
-  const imageId = randomImg(collection.objectIDs || [291826]);
+  const noImage = [291826];
```
Comment (kddrake) [inline src/api/get-profile-image.js]: dig what you've got here for `noImage`, I'm just wondering if you want to have something displayed in the UI to alert the user that a fallback image is being used.

File: src/components/profile-form.html [hunk 2]
```diff
@@ -15,6 +16,10 @@
     </style>
     <form class="validate">
       <fieldset>
+
+        <met-image first-name="[[userData.firstName]]" img="[[userData.img]]">
+        </met-image>
```
Comment (kddrake) [inline src/components/profile-form.html]: super nit: you should have the closing `</met-image>` up with the opening `<met-image>`

File: src/api/get-profile-image.js (hunk 1 above)
Comment (reuelcj) [reply src/api/get-profile-image.js]: Yes. I meant to add something explaining what was going on in general and linking to the met... but forgot to.

**Pull request merged**